x-pack/docs/en/security/authentication/built-in-users.asciidoc

```diff
@@ -91,6 +91,14 @@ new bootstrap password:
 bin/elasticsearch-keystore add "bootstrap.password"
 ----------------------------------------------------
 
+Alternatively you can set the bootstrap password by passing the password to stdin.
+This can be useful for scripted or automated deployments.
+
+[source,shell]
+------------------------------------------------------------------------------------------
+bash -c 'printf <PASSWORD> | bin/elasticsearch-keystore add bootstrap.password -x -f -v'
+------------------------------------------------------------------------------------------
```

Comment:
If we're going to recommend `-f` then I think we should explicitly call out that this will replace an existing value.

Reply:
This makes sense.